scripts/setup_github_app.py

```
@@ -0,0 +1,240 @@
+        print(f"ERROR: Cannot read private key file: {e}")
+        return False
+
+
+def test_webhook_endpoint():
+    """Test if the webhook endpoint is accessible."""
+    try:
+        import requests
+
+        response = requests.get("http://localhost:8000/health", timeout=2)
+        if response.status_code == 200:
+            print("✓ Event Listener service is running")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)